CHANGELOG.md

```diff
@@ -27,6 +27,20 @@ Mapbox welcomes participation and contributions from everyone.
 * Limit the delayed network request maximum time in the scheduler task queue, and thus avoid excessive memory usage. ([#1105](https://github.com/mapbox/mapbox-maps-android/pull/1105))
 * Fill extrusion layer support for globe view. ([#1105](https://github.com/mapbox/mapbox-maps-android/pull/1105))
 * Increase priority of a renderer thread. ([#1105](https://github.com/mapbox/mapbox-maps-android/pull/1105))
+* Refine viewport plugin's public APIs. ([#1069](https://github.com/mapbox/mapbox-maps-android/pull/1069))
```

Comment:
since there is another entry with `Introduce viewport plugin.`, we can merge these two, wdyt @pengdev ?

Reply:
Yes, but let's include both PRs